src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -157,13 +174,15 @@ def extract_strings(self) -> Mapping[int, Set[str]]:
 
             metric_name = message["name"]
             metric_type = message["type"]
+            use_case_id = message["use_case_id"]
             org_id = message["org_id"]
             tags = message.get("tags", {})
 
             if not valid_metric_name(metric_name):
                 logger.error(
                     "process_messages.invalid_metric_name",
                     extra={
+                        "use_case_id": use_case_id,
```

Comment:
Let's log the error for the other case below as well (if not in the ACCEPTED_METRIC_TYPES)